Dockerfile

```diff
@@ -34,4 +34,4 @@ WORKDIR /parseable
 COPY --from=builder /bin/sh /bin/sh
```

Comment:
@nitisht any specific reason for adding shell. Is it because image is distroless, i have been using Dockerfile.debug, so just want to confirm ?

Reply:
Yes, because distroless doesn't have any other shell.

Reply:
Hmm, but do we need a shell ? I mean wasn't shell for ${hostname}, container and app PID should be same. 
I'lll check once if needed. If you see CMD will trigger the command itself.

Reply:
Sure, we can remove in that case